### ISSUE TITLE
Apps in app drawer must be tapped twice to open

### ISSUE THREAD

**Issue by exstntlmsnthrp** [MM2-0/Kvaesitso#352]
For some reason, since the newest update apps in my app drawer must be tapped twice in order to open. Tap once, nothing happens; tap twice, opens like expected. 

Phone: Pixel 7 Pro
Android version: 13
Kvaesitso version: 1.24.1
App source: GitHub

Here are my logs:

```
Device: cheetah
SDK version: 33
====================================
04-20 20:58:06.499  3512  3512 W OnBackInvokedCallback: OnBackInvokedCallback is not enabled for the application.
04-20 20:58:06.499  3512  3512 W OnBackInvokedCallback: Set 'android:enableOnBackInvokedCallback="true"' in the application manifest.
04-20 20:58:08.309  3512  3512 D CompatibilityChangeReporter: Compat change id reported: 171228096; UID 10289; state: ENABLED
04-20 20:58:10.117  3512  3660 D OpenGLRenderer: endAllActiveAnimators on 0xb4000076b8274360 (UnprojectedRipple) with handle 0xb4000075c82ca2e0
04-20 20:58:11.950  3512  3512 W OnBackInvokedCallback: OnBackInvokedCallback is not enabled for the application.
04-20 20:58:11.950  3512  3512 W OnBackInvokedCallback: Set 'android:enableOnBackInvokedCallback="true"' in the application manifest.
04-20 20:58:12.100  3512  3539 W uncher2.release: Cleared Reference was only reachable from finalizer (only reported once)
04-20 20:58:13.939  3512  3660 D OpenGLRenderer: endAllActiveAnimators on 0xb4000076b8281160 (UnprojectedRipple) with handle 0xb4000075c8280ea0
04-20 20:58:14.841  3512  3512 W OnBackInvokedCallback: OnBackInvokedCallback is not enabled for the application.
04-20 20:58:14.841  3512  3512 W OnBackInvokedCallback: Set 'android:enableOnBackInvokedCallback="true"' in the application manifest.
04-20 20:59:16.997  3512  3512 W OnBackInvokedCallback: OnBackInvokedCallback is not enabled for the application.
04-20 20:59:16.997  3512  3512 W OnBackInvokedCallback: Set 'android:enableOnBackInvokedCallback="true"' in the application manifest.
04-20 20:59:20.621  3512  3512 W OnBackInvokedCallback: OnBackInvokedCallback is not enabled for the application.
04-20 20:59:20.621  3512  3512 W OnBackInvokedCallback: Set 'android:enableOnBackInvokedCallback="true"' in the application manifest.
04-20 20:59:20.688  3512  3660 D OpenGLRenderer: endAllActiveAnimators on 0xb4000076b82c57c0 (UnprojectedRipple) with handle 0xb4000075c8293800
04-20 20:59:38.072  3512  3512 W OnBackInvokedCallback: OnBackInvokedCallback is not enabled for the application.
04-20 20:59:38.072  3512  3512 W OnBackInvokedCallback: Set 'android:enableOnBackInvokedCallback="true"' in the application manifest.
04-20 21:00:13.578  3512  8659 E DatabaseUtils: Writing exception to parcel
04-20 21:00:13.578  3512  8659 E DatabaseUtils: java.lang.SecurityException: Permission Denial: reading de.mm20.launcher2.content.GenericFileProvider uri content://de.mm20.launcher2.release.fileprovider/external_files/Android/data/de.mm20.launcher2.release/files/kvaesitso-log-2023-04-20-210013 from pid=29767, uid=1000 requires the provider be exported, or grantUriPermission()
04-20 21:00:13.578  3512  8659 E DatabaseUtils: 	at android.content.ContentProvider.enforceReadPermissionInner(ContentProvider.java:879)
04-20 21:00:13.578  3512  8659 E DatabaseUtils: 	at android.content.ContentProvider$Transport.enforceReadPermission(ContentProvider.java:707)
04-20 21:00:13.578  3512  8659 E DatabaseUtils: 	at android.content.ContentProvider$Transport.query(ContentProvider.java:246)
04-20 21:00:13.578  3512  8659 E DatabaseUtils: 	at android.content.ContentProviderNative.onTransact(ContentProviderNative.java:107)
04-20 21:00:13.578  3512  8659 E DatabaseUtils: 	at android.os.Binder.execTransactInternal(Binder.java:1280)
04-20 21:00:13.578  3512  8659 E DatabaseUtils: 	at android.os.Binder.execTransact(Binder.java:1244)
04-20 21:00:13.594  3512  8659 E DatabaseUtils: Writing exception to parcel
04-20 21:00:13.594  3512  8659 E DatabaseUtils: java.lang.SecurityException: Permission Denial: reading de.mm20.launcher2.content.GenericFileProvider uri content://de.mm20.launcher2.release.fileprovider/external_files/Android/data/de.mm20.launcher2.release/files/kvaesitso-log-2023-04-20-210013 from pid=29767, uid=1000 requires the provider be exported, or grantUriPermission()
04-20 21:00:13.594  3512  8659 E DatabaseUtils: 	at android.content.ContentProvider.enforceReadPermissionInner(ContentProvider.java:879)
04-20 21:00:13.594  3512  8659 E DatabaseUtils: 	at android.content.ContentProvider$Transport.enforceReadPermission(ContentProvider.java:707)
04-20 21:00:13.594  3512  8659 E DatabaseUtils: 	at android.content.ContentProvider$Transport.query(ContentProvider.java:246)
04-20 21:00:13.594  3512  8659 E DatabaseUtils: 	at android.content.ContentProviderNative.onTransact(ContentProviderNative.java:107)
04-20 21:00:13.594  3512  8659 E DatabaseUtils: 	at android.os.Binder.execTransactInternal(Binder.java:1280)
04-20 21:00:13.594  3512  8659 E DatabaseUtils: 	at android.os.Binder.execTransact(Binder.java:1244)
04-20 21:00:13.628  3512  8659 E DatabaseUtils: Writing exception to parcel
04-20 21:00:13.628  3512  8659 E DatabaseUtils: java.lang.SecurityException: Permission Denial: reading de.mm20.launcher2.content.GenericFileProvider uri content://de.mm20.launcher2.release.fileprovider/external_files/Android/data/de.mm20.launcher2.release/files/kvaesitso-log-2023-04-20-210013 from pid=29767, uid=1000 requires the provider be exported, or grantUriPermission()
04-20 21:00:13.628  3512  8659 E DatabaseUtils: 	at android.content.ContentProvider.enforceReadPermissionInner(ContentProvider.java:879)
04-20 21:00:13.628  3512  8659 E DatabaseUtils: 	at android.content.ContentProvider$Transport.enforceReadPermission(ContentProvider.java:707)
04-20 21:00:13.628  3512  8659 E DatabaseUtils: 	at android.content.ContentProvider$Transport.query(ContentProvider.java:246)
04-20 21:00:13.628  3512  8659 E DatabaseUtils: 	at android.content.ContentProviderNative.onTransact(ContentProviderNative.java:107)
04-20 21:00:13.628  3512  8659 E DatabaseUtils: 	at android.os.Binder.execTransactInternal(Binder.java:1280)
04-20 21:00:13.628  3512  8659 E DatabaseUtils: 	at android.os.Binder.execTransact(Binder.java:1244)
null
```

**Comment by Sir-Photch** [comment]
I thought this was just me, but yes, this also happens on my device.

**Comment by exstntlmsnthrp** [comment]
> I thought this was just me, but yes, this also happens on my device.

It's exactly this why I decided to help however I can with open source projects, even though I'm not a developer or know anything about programming. I used to think problems were just me, then slowly find out others have them too.

**Comment by MM2-0** [comment]
Seems to be related to the pager layout. I can reproduce it if the search gesture is set to swipe left or right. Swipe down is working fine.

**Comment by MM2-0** [comment]
Issue in library: https://issuetracker.google.com/issues/276738324

**Comment by Sir-Photch** [comment]
I think this is happening again, can anyone confirm?

**Comment by exstntlmsnthrp** [comment]
Yes, definitely happening again for me too. I just started noticing today after updating yesterday evening.

**Comment by MM2-0** [comment]
Should be fixed in 1.26.3
